crates/astria-proto/proto/buf.yaml

```diff
@@ -1,7 +1,7 @@
 version: v1
 name: buf.build/astria/astria
 deps:
-  - buf.build/tendermint/tendermint
+  - buf.build/penumbra-zone/penumbra
```

Comment:
This should pin the specific version of the penumbra proto

Reply:
isn't it pinned in the buf.lock?

Reply:
Not the same kind of pinning IMO. The lock file encodes code generation at the time buf was run, whereas pinning in the YAML shows author intent.